qa/sc_cert_submitter_secure_enclave.py

```diff
@@ -18,7 +18,7 @@
     Start 1 MC node and 1 SC node.
     Enable Secure Enclave.
     SC node manages keys 0,1,2.
-    Secure Enclave manages keys 4,5,6
+    Secure Enclave manages keys the remaining keys (from 3 up to MAX_NUMBER_OF_KEYS)
```

Comment:
index 3 is intentinally skipped here, so it should be "from 4 up to MAX_NUMBER_OF_KEYS"